src/index.ejs

```diff
@@ -7,6 +7,10 @@
   </head>
 
   <body>
+    <div class="inline-fourohtwo">
+      <div id="preact_root"></div>
```

Comment:
this is a global id that might be used by the host page.

Reply:
Yes, I agree. Removed it.